brainscore/benchmarks/imagenet_c.py

```diff
@@ -116,38 +116,31 @@ class Imagenet_C_Individual(BenchmarkBase):
     """
     Runs an individual ImageNet C benchmark, like "gaussian_noise_1"
     """
-
-    def __init__(self, identifier_suffix, noise_type):
-        identifier = f'dietterich.Hendrycks2019.{identifier_suffix}'
-        stimulus_set = brainscore.get_stimulus_set(identifier)
-        self._stimulus_set = stimulus_set
-        self._similarity_metric = Accuracy()
-        self._benchmark_name = identifier
-        self._noise_type = noise_type
+    def __init__(self, stimulus_set, noise_level, noise_type):
+        #stimulus_set = brainscore.get_stimulus_set(benchmark_name)
+        self.stimulus_set = stimulus_set[stimulus_set['noise_level']==noise_level]
+        self.noise_level = noise_level
+        self.noise_type = noise_type
+        self.benchmark_name = f'dietterich.Hendrycks2019.{noise_type}-{noise_level}'
+        self._similarity_metric = Accuracy() 
         ceiling = Score([1, np.nan], coords={'aggregation': ['center', 'error']}, dims=['aggregation'])
-        super(Imagenet_C_Individual, self).__init__(identifier=f"{identifier}-top1", version=1,
+        super(Imagenet_C_Individual, self).__init__(identifier='dietterich.Hendrycks2019-top1', version=1,
                                                     ceiling_func=lambda: ceiling,
-                                                    parent=f'dietterich.Hendrycks2019-{noise_type}-top1',
+                                                    parent='dietterich.Hendrycks2019-top1',
```

Comment:
I think we last spoke about a hierarchy
dietterich.Hendrycks2019-top1
- dietterich.Hendrycks2019-top1-noise
  - dietterich.Hendrycks2019-top1-noise_1
  - dietterich.Hendrycks2019-top1-noise_5
- dietterich.Hendrycks2019-top1-blur

etc

Do you want to change it to a flat hierarchy?

Reply:
I still haven't understood exactly how the parent business works. Can I just list whatever I want as a parent and the framework will implicitly construct the hierarchy? or does the parent need to be the identifier of another benchmark?

I made another push updating the parents, let me know if that makes sense.

Reply:
Pretty much yes -- the framework in this case for now includes me making the parent database entries, but once they're there it will take care of an arbitrarily deep hierarchy. No need to define the parent as another benchmark

Reply:
I commented with what I think the identifier / parents should be, assuming I understand the hierarchy right (`dietterich.Hendrycks2019 > dietterich.Hendrycks2019-{noise_category} > dietterich.Hendrycks2019-{noise_category}-{noise_type}`)